hydra_ros/src/backend/ros_backend_publisher.cpp

```diff
@@ -177,14 +178,17 @@ void RosBackendPublisher::publishPoseGraph(const DynamicSceneGraph& graph,
   }
 
   auto pose_graph = *dgraph.getPoseGraph(id_timestamps, false, true);
+  pose_graph.stamp_ns = stamp;
```

Comment:
Mostly for my understanding: Is this to differentiate between two different versions of the pose graph, i.e., optimized at different times? Is there any case where the latest node timestamp isn't the timestamp of the pose graph? Probably not the worst thing to stamp it explicitly, just trying to figure out if `getPoseGraph` was doing something incorrectly before or just not setting it.

Reply:
This is because getPoseGraph does not set the stamp_ns. Decided to explicitly set here instead of reasoning about whether to use the last pose or last vertex as stamp